backend/pom.xml

```diff
@@ -177,6 +177,10 @@
             <artifactId>awaitility</artifactId>
             <scope>test</scope>
         </dependency>
+        <dependency>
+            <groupId>io.quarkus</groupId>
+            <artifactId>quarkus-smallrye-fault-tolerance</artifactId>
+        </dependency>
```

Comment:
Nitpick: Could you please move this up to the `<!-- Quarkus -->` "section" of this file?

Reply:
Moved!